data/[bzr99]_s_ori_70.json

```diff
@@ -18,5 +18,16 @@
         {
             "other_name": "[BZR99] S Ori 70"
```

Comment:
At some point I'd like to swap the primary identifier to be S Ori 70 rather than [BZR99] S Ori 70 but that is a separate issue.

Reply:
#134